caffe2/python/ideep/convfusion_op_test.py

```diff
@@ -189,6 +190,34 @@ def test_convolution_sum_fusion(self, stride, pad, kernel, size,
             print(S0.flatten())
             print(np.max(np.abs(S1 - S0)))
             self.assertTrue(False)
+
+        # Auto fusion for Conv + Sum
+        workspace.ResetWorkspace()
```

Comment:
I still don't understand how this can be used in inference and for Conv+Sum fusion, output will rewrite input 'S0'. If you run this op twice, you will get 2 different results as you input has changed. Please check https://github.com/intel/ideep/issues/8 for more discussion.

Reply:
1. training_mode flag changed the conv operator API for user, especially in training mode. Instead, optimizeForIdeep gives choice to user. :)

2. Yes. You're right. The input will be rewritten. But if the input is produced by other operator, instead of directly loaded by user, it should be OK in this case. Will add the check for it.